languages/javalang/builder-renderer/commercetools-java-client/build.gradle

```diff
@@ -18,4 +18,8 @@ publishing {
 }
 
 dependencies {
+    compile project(':languages:javalang:builder-renderer:commercetools-java-base')
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
+    implementation("com.squareup.okhttp3:okhttp:4.0.1")
```

Comment:
Why do we use `implementation` here? Why don't we use `compile` as with the other dependencies?

Reply:
copy/paste from okhttp documentation, I will change it.